src/massive/munit/Assert.hx

```diff
@@ -226,4 +226,25 @@ class Assert
 	{
 		throw new AssertionException(msg, info);
 	}
+	
+	/**
+	 * Assert that an expectation was thrown. Can expect strings and non-strings.
+	 *
+	 * @param   code				a function which should throw an exception 
+	 * @return					  the exception that was thrown	  
+	 * @throws  AssertionException  if no expectation is thrown
+	 */
+	public static function throws(code:Dynamic, ?info:PosInfos):Dynamic
+	{
+		try
+		{
+			code();
+			fail("Expected exception wasn't thrown!", info);
+			return null; // needed to compile
+		}
+		catch (e:Dynamic)
```

Comment:
The inspo for this new method I believe comes from c#? From what I can see specifying the type of exception to be thrown is core to its usage, otherwise you always have to write two assertions. The first `Assert.throws..` and the second to assert that the correct thing was thrown. Only doing the former mean you could be catching the wrong exception and never know.